packages/typo3-guides-cli/src/Command/InitCommand.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\GuidesCli\Command;
+
+use T3Docs\GuidesCli\Generation\DocumentationGenerator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use T3Docs\VersionHandling\Packagist\ComposerPackage;
+use T3Docs\VersionHandling\Packagist\PackagistService;
+
+/**
+ * You can run this command, for example like
+ *
+ * ddev exec packages/typo3-guides-cli/bin/typo3-guides init --working-dir=packages/typo3-guides-cli
```

Comment:
I don't understand this example. Why should I use the typo3-guides directory as working dir?

```suggestion
 * ddev exec packages/typo3-guides-cli/bin/typo3-guides init --working-dir=packages/my-extension
```

Your example is for legacy installation. I would prefer to add a composer-based example, too.

Reply:
we are using composer, later on you will use that with the docker container but we will document that in "How to document"